client/src/components/App.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import $ from 'jquery';
+import styled from 'styled-components';
+import Carousel from './Carousel.jsx';
+
+const exampleData = require('../../../exampleData.js');
+
```

Comment:
Example data will eventually be replaced with real data from the ajax call, correct?

Reply:
correct!